assets/cart.js

```diff
@@ -100,6 +100,7 @@ class CartItems extends HTMLElement {
       .then((state) => {
         const parsedState = JSON.parse(state);
         const quantityElement = document.getElementById(`Quantity-${line}`) || document.getElementById(`Drawer-quantity-${line}`);
+        const items = document.querySelectorAll('.cart-item');
 
         if (parsedState.errors) {
```

Comment:
The same way the error of min shows, shouldnt the error for max show as well? 🤔 

https://screenshot.click/31-39-0dbqc-vsfu2.mp4

Reply:
Also after it changes to 84, the +/- dont seem to work anymore and keep defaulting back to 84

https://screenshot.click/31-41-o9daq-g2q1q.mp4

Reply:
Cart drawer was enabled good point 👍 

What about the min error? 👀 https://github.com/Shopify/dawn/pull/2251#discussion_r1092114873